internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -822,6 +822,41 @@ components:
           description: |
             Naming package sets for a repository assigns it to a specific part
             (pipeline) of the build process.
+    CustomRepository:
```

Comment:
You are missing the `filename` property in the definition.

Reply:
Ah yeah, causing a failure in CI